config/jobs/templates/README.md

```diff
@@ -0,0 +1,2 @@
+#### This Folder Is Literally Just Hacks for Mass Producing Jobs
```

Comment:
This is great

Reply:
I don't want to lie, but that is why you see these mass produced templates :). It is not long term appropriate of course but for sandboxing and ensuring I check every tick box, it sure is handy..